website/docs/installation/ferretdb/deb.md

```diff
@@ -21,13 +21,11 @@ You can check that FerretDB was installed by running
 ferretdb --version
 ```
 
+:::caution
 FerretDB does not automatically install PostgreSQL.
-To install PostgreSQL, run the following commands:
-
-```sh
-sudo apt update
-sudo apt install -y postgresql
-```
+The `.deb` package for PostgreSQL with DocumentDB extension is not currently available.
```

Comment:
This is really odd to read within :::caution::: block. The two sentences are not connected. And we say that deb package is not available half way down that page. Do we need this change? The same applies to rpm.